CONTRIBUTING.md

```diff
@@ -1,18 +1,18 @@
-# Contributing to Node.js
+# Contributing to Ayo.js
 
 ## Code of Conduct
 
 Please read the
 [Code of Conduct](https://github.com/nodejs/TSC/blob/master/CODE_OF_CONDUCT.md)
-which explains the minimum behavior expectations for Node.js contributors.
+which explains the minimum behavior expectations for Ayo.js contributors.
 
 ## Issue Contributions
 
 When opening issues or commenting on existing issues, please make sure
-discussions are related to concrete technical issues with Node.js.
+discussions are related to concrete technical issues with Ayo.js.
 
-* For general help using Node.js, please file an issue at the
-[Node.js help repository](https://github.com/nodejs/help/issues).
+* For general help using Ayo.js, please file an issue at the
+[ayo.js help repository](https://github.com/nodejs/help/issues).
```

Comment:
should we leave this at `node.js help repository`? otherwise it's a misguided link

Reply:
it seems a little confusing to continue linking back to node's help repo.. can we make the new repo now or is something preventing that?

Reply:
@Kingles we're doing small tasks and iterations right now to lay ground work. We're not ready to do the help repo just yet.

Reply:
We could just point it at this repo for now and change it to point at another repo if we make another at some point in the future.